tox.ini

```diff
@@ -2,7 +2,7 @@
 ; under all supported Python interpreters
 
 [tox]
-envlist = py27,py33,py34,py35
+envlist = py27,py33,py38
```

Comment:
do you want to keep py33 here? it is dropped in travis.yml

Reply:
It seems travis is not working in this repository. But in the fork, it is enabled, and even if Travis is configured to work with Python 3.8, Python 3.5 is used by Tox. See this run: https://github.com/scrapinghub/autopager/pull/1/checks?check_run_id=1081133825

I could remove python 3.5 or even python 2.7, but why not leave them given that they are working?